reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,476 @@
+      italic(
+        "- The collateral amount used to calculate GCR's is equal to the current collateral deposited, and does not include liquidated collateral"
+      )
+    );
+    await this._generateSponsorStats();
+    console.groupEnd();
+
+    // 2. Tokens stats table
+    console.group();
+    console.log(bold("Token summary stats"));
+    console.log(italic("- Token price is sourced from exchange where synthetic token is traded (i.e. Uniswap)"));
+    console.log(
+      italic(
+        "- Uniswap TWAP price window can be modified using the 'twapLength' property in the UNISWAP_PRICE_FEED_CONFIG"
+      )
+    );
+    await this._generateTokenStats();
+    console.groupEnd();
+
+    // 3. Liquidation stats table
+    console.group();
+    console.log(bold("Liquidation summary stats"));
+    console.log(italic("- Unique liquidations count # of unique sponsors that have been liquidated"));
+    console.log(italic("- Collateral & tokens liquidated counts aggregate amounts from all partial liquidations"));
+    console.log(italic("- Current collateral liquidated includes any collateral locked in pending liquidations"));
+    await this._generateLiquidationStats();
+    console.groupEnd();
+
+    // 4. Dispute stats table
+    console.group();
+    console.log(bold("Dispute summary stats"));
+    await this._generateDisputeStats();
+    console.groupEnd();
+
+    // 5. DVM stats table
+    console.group();
+    console.log(bold("DVM summary stats"));
+    await this._generateDvmStats();
+    console.groupEnd();
+  };
+
+  _generateSponsorStats = async () => {
+    const { toBN, toWei } = this.web3.utils;
+
+    let allSponsorStatsTable = {};
+
+    if (this.newSponsorEvents.length === 0) {
+      console.log(dim("\tNo positions have been created for this EMP."));
+      return;
+    }
+
+    // - Lifetime # of unique sponsors.
+    const uniqueSponsors = {};
+    const currentUniqueSponsors = this.empClient.getAllPositions();
+    for (let event of this.newSponsorEvents) {
+      uniqueSponsors[event.sponsor] = true;
+    }
+    allSponsorStatsTable["# of unique sponsors"] = {
+      cumulative: Object.keys(uniqueSponsors).length,
+      current: Object.keys(currentUniqueSponsors).length
+    };
+
+    // - Cumulative collateral deposited into contract
+    let collateralDeposited = toBN("0");
+    let collateralDepositedPeriod = toBN("0");
+    for (let event of this.collateralDepositEvents) {
+      collateralDeposited = collateralDeposited.add(toBN(event.returnValues.value));
+      if (event.blockNumber >= this.startBlockNumberForPeriod) {
+        collateralDepositedPeriod = collateralDepositedPeriod.add(toBN(event.returnValues.value));
+      }
+    }
+    allSponsorStatsTable["collateral deposited"] = {
+      cumulative: this.formatDecimalString(collateralDeposited),
+      [this.periodLabelInHours]: this.formatDecimalString(collateralDepositedPeriod),
+      current: this.formatDecimalString(this.totalPositionCollateral)
```

Comment:
Just want to clarify, this is intended to *only* be the *live* positions (not collateral that's currently undergoing liquidation)? Liquidation collateral is *not* excluded in the cumulative or period sections.

Reply:
exactly this is the `totalPositionCollateral()` result

Liquidation collateral is not excluded in cumulative/period sections because liquidations do not reflect a `collateralCurrency.Transfer` event. This is why it is possible for the `current collateral deposited` to differ from the `net collateral deposited` iff there is a pending liquidation.

Reply:
Yeah, I think I get that. I guess `current` could have a lot of interpretations, but "current live deposits" is probably the one that makes the most sense.

Reply:
Agreed. I wanted to pick a column label that could apply to the rest of the rows. See the attached screengrab